js/ui/map.js

```diff
@@ -91,38 +92,45 @@ util.extend(Map.prototype, {
     },
 
     // Set the map's center, zoom, and bearing
-    setView: function(center, zoom, bearing) {
+    setView: function(center, zoom, bearing, pitch) {
         this.stop();
 
         var tr = this.transform,
             zoomChanged = tr.zoom !== +zoom,
-            bearingChanged = tr.bearing !== +bearing;
+            bearingChanged = tr.bearing !== +bearing,
+            pitchChanged = tr.pitch !== +pitch;
```

Comment:
Yeah, that would be way more convenient https://github.com/mapbox/mapbox-gl-js/issues/1051